File: analyzer/project/sql/pg_client.py
import os
import psycopg2
from dotenv import load_dotenv

load_dotenv()

connection = psycopg2.connect(user=os.getenv("DB_USER"),
                              host=os.getenv("DB_HOST"),
                              port=os.getenv("DB_PORT"),
                              database=os.getenv("DB_DATABASE"))


def execute_sql(query, interpolation_args={}):
    built_query = query.format(**interpolation_args)

    print(built_query)

    cursor = connection.cursor()
    cursor.execute(built_query)
    results = cursor.fetchall()
    cursor.close()
    return results

File: seeder/pg-client.js
require('dotenv').config();

const { Client }        = require('pg');

const enableSqlLogging  = process.env.SQL_LOGGING;

const client = new Client({
  user:     process.env.DB_USER,
  host:     process.env.DB_HOST,
  database: process.env.DB_DATABASE,
  port:     process.env.DB_PORT,
});

client.connect();

async function executeSql(statement) {
  if (enableSqlLogging) {
    console.debug(statement);
  }

  return client.query(statement)
    .catch((err) => {
      console.error('ERROR', err);
    });
}

module.exports = {
  executeSql,
};
